operator/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Main
 
+- [5588](https://github.com/grafana/loki/pull/5588) **periklis**: Fix prometheus scrape access to Loki component metrics
```

Comment:
Should we update the name of this PR to reflect that this is more about the SD rather than the scrape?

```suggestion
- [5588](https://github.com/grafana/loki/pull/5588) **periklis**: Create role for Prometheus service discovery to Loki component metrics
```